addons/fastroping/script_component.hpp

```diff
@@ -16,3 +16,8 @@
 #endif
 
 #include "\z\ace\addons\main\script_macros.hpp"
+
+
+#define ANIMS_HOOK ["extendHookRight", "extendHookLeft"]
+#define ANIMS_DOOR ["door_R", "door_L", "CargoRamp_Open", "Door_rear_source", "Door_6_source", "CargoDoorR", "CargoDoorL"]
+#define ANIMS ["dvere1_posunZ", "dvere2_posunZ", "doors"]
```

Comment:
Could we find a different name for this macro? "Anims" is so generic.

Reply:
`ANIMS_DOOR` -> `animateDoor` (requires "door" source)
`ANIMS` -> `animate` ("user" source)

Reply:
The only reason they are split in two is because they require different commands (above comment).

I have no idea what else to name it, suggestions?

Reply:
`ANIMS_USER` then?

Reply:
I named them `ANIMS_ANIMATEDOOR` and `ANIMS_ANIMATE` to make clear what command is which. Is that good or should I go with `ANIMS_DOOR` and `ANIMS_USER` rather? One is what command is used, the other is what type is used in config.